.travis.yml

```diff
@@ -33,9 +33,9 @@ matrix:
     - php: 7.1
       env: TEST_CONFIG="phpunit.xml"
     - php: 7.1
-      env: REST_TEST_CONFIG="phpunit-functional-rest.xml" RUN_INSTALL=1 SYMFONY_ENV=behat SYMFONY_DEBUG=1
+      env: REST_TEST_CONFIG="phpunit-functional-rest.xml" SYMFONY_ENV=behat SYMFONY_DEBUG=1 SF_CMD="ez:behat:create-language 'pol-PL' 'Polish (polski)'"
```

Comment:
this should be applied on 6.13 as well right? or is there some other reason why this is needed now and was not before?

Reply:
It already [is present on `6.13`](https://github.com/ezsystems/ezpublish-kernel/blob/6.13/.travis.yml#L41) and `master`, but was missing on `7.0`. 
However `7.0` has merged in PR introducing this, so I wondered myself how we lost that one. Maybe some conflict fix during rebase?

Reply:
either that or we have not merged it up yet

Reply:
just make sure they are aligned and we avoid conflicts when we sync them up.